### PR TITLE
[#7371] Add curl to development package dependencies (main)

### PR DESCRIPTION
**Pull request by SwooshyCueb**
Addresses #7371

### CODE REVIEW

File: cmake/packaging.cmake [hunk 1]
```diff
@@ -54,40 +54,33 @@ endif()
 
 set(CPACK_ARCHIVE_COMPONENT_INSTALL ON)
 
+# identify proper curl packages for debian
```
Comment (alanking) [inline cmake/packaging.cmake:57]: Is "debian" referring to the distro or the package format (which would include Ubuntu)?

**Review by alanking — COMMENTED** [review]
Just had one clarifying question. Seems fine

File: cmake/packaging.cmake (hunk 1 above)
Comment (SwooshyCueb) [reply cmake/packaging.cmake:57]: package format

**Review by alanking — APPROVED** [review]
Seems good to me. I think we can go ahead and # this.

**Comment by SwooshyCueb** [comment]
\#'d

**Pull request merged**